db.py

```diff
@@ -0,0 +1,29 @@
+import psycopg2
+import os
+
+
+def instantiate_db_connection():
+    """Returns the connection from the DB"""
+
+    db_uri = os.getenv("DB_URI")
+    return psycopg2.connect(db_uri)
+
+
+def add_column(column_name: str, data_type: str) -> None:
+    """Adds a column for adjustment to the table after the table has been created"""
+
+    with instantiate_db_connection() as connection:
+        cursor = connection.cursor()
+
+        cursor.execute(f"ALTER TABLE jobs_table ADD COLUMN {column_name} {data_type}")
+
+        connection.commit()
+
+
+def delete_alL_opportunity_type(opp_type: str) -> None:
```

Comment:
Where is this function used? 
```suggestion
def delete_all_opportunity_type(opp_type: str) -> None:
```

Reply:
This is just a function for debugging, I found it useful to keep it there just in case for anything.

Reply:
> This is just a function for debugging, I found it useful to keep it there just in case for anything.

Feel free to mention this in the documentation comment of that function signature.